<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite hostname="E5470-01381" ignored="0" name="TestA1" tests="3" failures="1" timestamp="23 Apr 2018 15:08:33 GMT" time="0.043" errors="0">
  <testcase name="TestA11" time="0.015" classname="suiteA.TestA1"/>
  <testcase name="TestA12" time="0.002" classname="suiteA.TestA1">
    <failure type="java.lang.AssertionError" message="expected [B] but found [A]">
      <![CDATA[java.lang.AssertionError: expected [B] but found [A]
at suiteA.TestA1.TestA12(TestA1.java:19)
at java.util.concurrent.FutureTask.run(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor.runWorker(Unknown Source)
at java.util.concurrent.ThreadPoolExecutor$Worker.run(Unknown Source)
at java.lang.Thread.run(Unknown Source)
... Removed 22 stack frames]]>
    </failure>
  </testcase> <!-- TestA12 -->
  <testcase name="TestA13" time="0.0" classname="suiteA.TestA1">
    <skipped/>
  </testcase> <!-- TestA13 -->
</testsuite> <!-- TestA1 -->
